test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -229,6 +304,32 @@ class test_ucp_sockaddr : public ucp_test {
         }
     }
 
+    void wait_for_server_reject(bool wakeup)
+    {
```

Comment:
what does reject on the server side mean? when would it happen?

Reply:
`ucp_test_base::entity::reject_ep_addr_cb` is called

Reply:
so wait_for_server_reject means that the server will _do_ the reject,
and wait_for_client_reject means that the client will _receive_ the reject from the server?